frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueCard.tsx

```diff
@@ -56,9 +56,14 @@ export function MissionQueueCard({ mission, onDeleteMission }: MissionQueueCardP
             <HorizontalContent>
                 <HorizontalNonButtonContent onClick={routeChange}>
                     <Checkbox />
-                    <Button variant="ghost" fullWidth>
-                        <Typography variant="body_short_bold">{mission.name}</Typography>
-                    </Button>
+
+                    {mission.name === 'dummyMission' ? (
```

Comment:
Since dummyMission is exported, can we not just check mission === dummyMission? This would prevent naming conflicts.

Reply:
agree, fixed